compliance/election.go

```diff
@@ -132,6 +132,49 @@ func TestDifferingElectionParameters(c *fluent.GRIBIClient, t testing.TB, opts .
 	)
 }
 
+// TestParamsDifferFromOtherClients checks that when a client A is connected with election ID of 10
```

Comment:
(nit) the key thing here is that client A is connected as a `SINGLE_PRIMARY` client, and client B wants to be an `ALL_PRIMARY` client. `ALL_PRIMARY` means that the server (device) itself should accept routes from different clients and select between them, whereas the `SINGLE_PRIMARY` mode means that there is an election ID that is supplied from some external source of elections which designates a single client as the primary, who then injects routes - and the others connected clients do not.

I'd suggest just clarifying here by saying "when client A is connected with `SINGLE_PRIMARY` mode...` replacing the "with election ID" part.

Reply:
Acknowledged! Changed the description as per the suggestions.